### PR TITLE
Fixes to tests for a green build

### PR DESCRIPTION
**Pull request by mattcan**
##### Description

This is an attempt to bring the coverage up on **assemble_leaderboards.py** as well as fix the red build.

Notes:

* I left `do_leaderboard_feed` out because I was running into errors. I figure it is more important to get the build green and open a separate issue for `do_leaderboard_feed`.
* I'm honestly not too sure why the `DAILY_CUTOFF` has to be offset 25 hours instead of 24, I think I lack the understanding of what the underlying data looks like.

##### Refers/Fixes

Refers: #8829 
Refers: #8862 

##### Testing

Tests pass :D

### CODE REVIEW

**Comment by mattcan** [comment]
Comments from #dc46c75:
> The fix in 8829 originally worked, when the code was submitted on Apr
28. When the PR was merged, the build actually failed. The reason for
this is that `timezone.now` was not patched prior to the `*_CUTOFF`
constants being set. This change proves that manipulating the
`DAILY_CUTOFF` we can get a passing test.

I'm going to come back tonight/tomorrow morning to poke at this and make it cleaner. I'm a bit slow having not done Python in almost 10 years

File: app/git/tests/test_utils.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def test_reset_token(self):
         """Test the github utility reset_token method."""
         auth_dict = build_auth_dict()
         url = TOKEN_URL.format(**auth_dict)
-        data = {'access_token': self.user_oauth_token}
+        data = {'token': self.user_oauth_token}
```
Comment (gdixon) [inline app/git/tests/test_utils.py:100]: Nice catch!! 🚀

File: app/marketing/tests/management/commands/test_assemble_leaderboards.py [hunk 2]
```diff
@@ -23,10 +23,11 @@
 from django.contrib.contenttypes.models import ContentType
 from django.utils import timezone
 
-from dashboard.models import Bounty, BountyFulfillment, Earning, Profile, Tip, UserAction
+import pytest
+from dashboard.models import Activity, Bounty, BountyFulfillment, Earning, Profile, Tip, UserAction
 from marketing.management.commands import assemble_leaderboards
 from marketing.management.commands.assemble_leaderboards import (
-    BREAKDOWNS, do_leaderboard, run_monthly, run_quarterly, run_weekly, run_yearly,
+    BREAKDOWNS, DAILY, do_leaderboard, run_monthly, run_quarterly, run_weekly, run_yearly, should_suppress_leaderboard,
```
Comment (gdixon) [inline app/marketing/tests/management/commands/test_assemble_leaderboards.py:30]: Can we remove `DAILY` - it's not being used 👍

File: app/marketing/tests/management/commands/test_assemble_leaderboards.py [hunk 3]
```diff
@@ -38,6 +39,7 @@ class TestAssembleLeaderboards(TestCase):
 
     def setUp(self):
         """Perform setup for the testcase."""
+        patch('marketing.management.commands.assemble_leaderboards.DAILY_CUTOFF', datetime(2021, 4, 27, 10, 00, tzinfo=UTC)).start()
```
Comment (gdixon) [inline app/marketing/tests/management/commands/test_assemble_leaderboards.py:42]: Awesome!!

**Review by gdixon — APPROVED** [review]
Great work @mattcan, thanks for this!

File: app/marketing/tests/management/commands/test_assemble_leaderboards.py (hunk 2 above)
Comment (mattcan) [reply app/marketing/tests/management/commands/test_assemble_leaderboards.py]: Good catch, change pushed

Comment (gdixon) [reply app/marketing/tests/management/commands/test_assemble_leaderboards.py]: Awesome, thanks Matt!

**Review by thelostone-mc — COMMENTED** [review]
LGTM

**Pull request merged**